device-sdks/index.html

```diff
@@ -0,0 +1,2205 @@
+        <p class="sdk-paragraph">
+          This feature is not available yet on ESP32 and is WIP on Rust and
+          Python.
+        </p>
+        <h2 id="device-registration" class="sdk-heading section-heading">
+          <a href="#device-registration" class="hover-link"
+            ><span class="icon-link" aria-hidden="true"></span
+          ></a>
+          Device Registration
+        </h2>
+        <p class="sdk-paragraph">
+          A device must be registered beforehand to obtain its
+          <code class="code-inline">credentials-secret</code>. While there are some
+          manual options (such as using the
+          <a class="link" href="https://github.com/astarte-platform/astartectl"
+            ><code class="inline">astartectl</code></a
+          >
+          command or using the
+          <a class="link" href="../latest/015-astarte_dashboard.html"
+            ><code class="inline">Astarte Dashboard</code></a
+          >), almost all Astarte Device SDKs allow to programmatically register
+          a Device. For Go you can use the
+          <a class="link" href="https://github.com/astarte-platform/astarte-go"
+            >astarte_go</a
+          >
+          client.
+        </p>
+      </div>
+    </section>
+
+    <div class="sdk-container">
+      <div class="nav-container">
+        <div
+          class="active underline sdk-item"
+          id="C"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          C
+        </div>
+        <div
+          class="underline sdk-item"
+          id="C++"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          C++
+        </div>
+        <div
+          class="underline sdk-item"
+          id="Elixir"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          Elixir
+        </div>
+        <div
+          class="underline sdk-item"
+          id="Go"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          Go
+        </div>
+        <div
+          class="underline sdk-item"
+          id="Java"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          Java/Android
+        </div>
+        <div
+          class="underline sdk-item"
+          id="Python"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          Python
+        </div>
+        <div
+          class="underline sdk-item"
+          id="Rust"
+          onclick="populateData(id)"
+          style="cursor: pointer"
+        >
+          Rust
+        </div>
+      </div>
+      <div class="sdk-description">
+        <div id="content"></div>
+      </div>
+    </div>
+
+    <script>
+      var targetDiv = document.getElementById("content");
+      var listIds = ["C", "C++", "Elixir", "Go", "Java", "Python", "Rust"];
+
+      var htmlCElement = `<section class="sdk-intro-container">
```

Comment:
This is a HUGE chunk of code embedded in a string. This makes it quite hard to maintain it.
Normally i would recommend splitting this in multiple files, but since we're using plain HTML, i'd rather have all the content already in place and just show/hide the appropriate block with `display: none`

Reply:
Fixed, I did it as you recommend.